.project

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@damosvil does it make sense to commit this file? Seems like it's just the default .project file? If it does not help with setup, I am not sure if we should keep it?

Reply:
Removed and added it to .gitignore